### PR TITLE
[v9] Add in version string definition for role in Terraform reference

### PR DESCRIPTION
**Pull request by stevenGravy**
backport #10609

### CODE REVIEW

**Review by klizhentas — APPROVED** [review]

File: docs/pages/setup/reference/terraform-provider.mdx [hunk 1]
```diff
@@ -167,6 +167,9 @@ Metadata specifies a resource name, description, labels and optional expiry date
 |              `namespace`|     string |          | Resource namespace ("default" by default) |
 |                `expires`|     string |          | Expiry time in RFC8829 format             |
 
+**version**
+
+`version` is an optional string that specifies the role version (default `v3`).
```
Comment (klizhentas) [inline docs/pages/setup/reference/terraform-provider.mdx:172]: isn't default a v4 now?

Comment (klizhentas) [reply docs/pages/setup/reference/terraform-provider.mdx:172]: @stevenGravy may be make it a variable instead of hardcode?

Comment (r0mant) [reply docs/pages/setup/reference/terraform-provider.mdx:172]: In Teleport 9 the default is `v5` I believe. @xacrimon Can you confirm?

Comment (xacrimon) [reply docs/pages/setup/reference/terraform-provider.mdx:172]: Yup

Comment (stevenGravy) [reply docs/pages/setup/reference/terraform-provider.mdx:172]: so this is still accurate in that it's the terraform, not a sample

Comment (stevenGravy) [reply docs/pages/setup/reference/terraform-provider.mdx:172]: @klizhentas I think you could make it variable.  this was the reference so just wanted it documented that we have this as a string

**Review by ptgott — APPROVED** [review]

**Pull request merged**